code/game/gamemodes/dynamic/dynamic_maths.dm

```diff
@@ -0,0 +1,48 @@
+// Exponential distribution.
+// Used for midround/latejoin injection delays.
+/proc/exp_distribution(var/desired_mean)
+	if (desired_mean <= 0)
+		desired_mean = 1
```

Comment:
I feel you shouldn't be hiding potential errors from people passing in 0 to this